src/css/_sets-manager.scss

```diff
@@ -87,12 +88,13 @@
                 z-index: 1;
                 padding-top: 1.25rem;
                 margin-top: -1.25rem;
-                border-bottom: 1px solid $color-gray-mid;
-                background-color: #222;
                 /* Need calculation, since with fixed position,
                     does not take padding into account. */
                 width: calc(100% - 3.75rem);
                 position: fixed;
+                border-bottom: 1px solid #555555;
```

Comment:
Here and below, if colors wouldn't change, and it's intentional, maybe add a note so no one worries later?